src/database/Database.cpp

```diff
@@ -263,10 +262,10 @@ Database::applySchemaUpgrade(unsigned long vers)
         {
             // Add columns for the LedgerEntry extension to each of
             // the tables that stores a type of ledger entry.
-            addTextColumn("accounts", ledgerExtName);
-            addTextColumn("trustlines", ledgerExtName);
-            addTextColumn("accountdata", ledgerExtName);
-            addTextColumn("offers", ledgerExtName);
+            addTextColumn("accounts", "ledgerext");
```

Comment:
it looks like this commit also contains changes for the previous commit (cleanup?)

Reply:
Yep, commits have been cleaned up.